### PR TITLE
Composant favori menu deroulant

### PR DESCRIPTION
**Pull request by Aristotia**
FavouriteCard.jsx, DisplayFavouriteCard.jsx, DisplayFavouriteCard.css created;
FavouriteCard added to SlideForCards;
JaugeVelo in construction

### CODE REVIEW

File: frontend/src/components/DisplayFavouriteCard.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.CardStationComp {
```
Comment (AnasMama) [inline frontend/src/components/DisplayFavouriteCard.css:1]: class en kebab case

File: frontend/src/components/DisplayFavouriteCard.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./DisplayFavouriteCard.css";
+
+export default function DisplayFavouriteCard({ favouriteCard }) {
+  console.log(favouriteCard);
+  return (
+    <div className="CardStationComp">
```
Comment (AnasMama) [inline frontend/src/components/DisplayFavouriteCard.jsx:7]: class en kebab case

**Review by MidNightRoad — COMMENTED** [review]
yes

**Review by AnasMama — APPROVED** [review]
Ok pour moi

**Pull request merged**